samples/embedded-auth-with-sdk/src/test/java/info/seleniumcucumber/userStepDefinitions/Login.java

```diff
@@ -83,6 +83,17 @@ public void refresh_token_stored() {
 		Assert.assertFalse("Can't access refresh_token", refreshToken.isEmpty());
 	}
 
+	@And("the cell for the value of \"email\" is shown and contains her {word}")
+	public void the_cell_for_email_is_shown_and_contains_her_email(String email) {
+		Assert.assertTrue(rootPage.email.isDisplayed());
+		Assert.assertEquals(System.getenv(email), rootPage.email.getText());
+	}
+
+	@And("^the cell for the value of \"name\" is shown and contains her first name and last name$")
+	public void the_cell_for_name_is_shown_and_contains_her_name() {
+		Assert.assertTrue(rootPage.name.isDisplayed());
```

Comment:
We should also validate the other half of this statement. `and contains her first name and last name`